lib/utils/getimage.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:pokedex/utils/strings.dart' as str;
+import 'package:pokedex/utils/constant.dart' as k;
+
+extension GetImage on String {
+  String getImage() => k.imageURL.replaceAll(str.indexReplace, split('/')[6]);
```

Comment:
what you did here is not to get image right but to manipulate its url transformation? then maybe rename this based on it's purpose like => `toCustomUrl` => then have it as a `getter` function not a regular new function so it will now look like this => `pokemon.url.toCustomUrl`